scripts/check_compliance.py

```diff
@@ -134,15 +187,13 @@ def run(self):
         self.prepare()
 
         if not self.zephyr_base:
-            self.case.result = Skipped("Not a Zephyr tree", "skipped")
-            return
+            self.skip("Not a Zephyr tree (ZEPHYR_BASE unset)")
 
         # Put the Kconfiglib path first to make sure no local Kconfiglib version is
         # used
         kconfig_path = os.path.join(self.zephyr_base, "scripts", "kconfig")
         if not os.path.exists(kconfig_path):
-            self.case.result = Error("Can't find Kconfig", "error")
-            return
+            self.error(kconfig_path + " not found")
```

Comment:
The new code changes the behavior here: it moves the text from the Error(msg=text, ...) constructor to ._elem.text. Is this intended and will github/JUnit be happy with this?

(basically the same discussion than the Skip() change you just did)

Reply:
Think it'll be fine. I think the constructor argument is more like a short summary/title, which is always shown, while the `_text` stuff is the body text in e.g. the GitHub comment and on Shippable.

Reply:
Oh... wait, this does put it in the `_text`. Confused by my own helpers.

Yeah, maybe it should be moved. I'll do a test commit to check how it renders.

Reply:
Seems you get neither the constructor argument nor the body text anywhere, on GitHub at least. Just a "Error during verification, please report!".

I dropped a random `self.error("yada yada")` at the start of a test.

Reply:
I changed `error()` to work the same as `skip()` (will get a really long "title" if there are any failures prior, but `skip()` has that problem too).

Looks like the `error()` message might not show up anywhere on GitHub anyway (dunno about Shippable), so that might need to be fixed separately.

Would really help to be able to check Shippable output in these PRs.